dune/xt/functions/base/combined-element-functions.hh

```diff
@@ -336,22 +336,22 @@ public:
 template <class LeftType, class RightType>
 struct DualStorageProvider
 {
-  XT::Common::StorageProvider<LeftType> left;
-  XT::Common::StorageProvider<RightType> right;
+  XT::Common::StorageProvider<LeftType> left_;
```

Comment:
Those were actually on purpose. Whenever we have a struct or public members of a class, I drop the trailing underscore, since those can be accessed from the outside directly. Opinions, @renefritze?

Reply:
I remember there were some warnings about something being shadowed, but that can probably be fixed differently. I'm okay with removing the underscores for public members.

Reply:
I removed the trailing underscores again.